### ISSUE TITLE
Download datasets

### ISSUE THREAD

**Issue by ngovanmao** [FCCLab/Neurosurgeon-chinese-rebuild#8]
- [x] ImageNet: The dataset includes images of 1000 classes, and is split into three sets: training (1.3M images), validation (50K images), and testing (100K images with held-out class labels). The classification performance is evaluated using two measures: the top-1 and **top-5 error** (main metric). We can use validation set as test set in this task.
- [ ] COCO:
